src/test/resources/product-json-schema.json

```diff
@@ -73,7 +73,6 @@
     },
 
     "Value": {
-      "type": "object",
```

Comment:
Maybe it is a good idea to collect answer to common questions in some place to provide link in the future in order not explain the same things many times.